v15-upgrade-log.md

```diff
@@ -32,3 +32,9 @@ Now we will upgrade to SpatialOS 15.0.0.
 1. Address changes made to command request op metadata. As we have deleted the
    code for dynamically giving authority to a client, we only need to change
    some logging messages for now.
+1. Add component sets. The physics worker always had authority over the EntityAcl,
+   the Position, and the Login component, while the client had authority over the
+   ClientData component. We will therefore need two component sets to represent
+   the two different units of authority in our project. We could replace the
+   EntityAcl with the AuthorityDelegation component, but with the new user-space
```

Comment:
Is there a way we can link to any more information about user-space loadbalancing? Or is this just something we can't do given we might be making those sorts of docs internal or something?